integrations/visual_studio/creating_packages.rst

```diff
@@ -0,0 +1,245 @@
+.. _visual_studio_packages:
+
+|visual_logo| Creating and reusing packages based on Visual Studio
+==================================================================
+
+Conan has different helpers to manage Visual Studio and MSBuild based projects.
+This how-to illustrates how to put them together to create and consume packages that are purely
+based on Visual Studio. This how-to is using VS2015, but other versions can be used too.
+
+
+Creating packages
+------------------
+
+Start cloning the existing example repository, containing a simple "Hello World" library, and
+application:
+
+.. code-block:: bash
+
+    $ git clone https://github.com/memsharded/hello_vs
```

Comment:
There is work to do here, because this is in my personal repo. Must be moved to "examples" repo first